cmd/derper/derper.go

```diff
@@ -155,7 +155,7 @@ func writeNewConfig() config {
 func checkMeshKey(key string) (string, error) {
 	key = strings.TrimSpace(key)
 	if !validMeshKey.MatchString(key) {
-		return "", fmt.Errorf("key in %q must contain 64+ hex digits", key)
+		return "", fmt.Errorf("key contain 64 hex digits")
```

Comment:
I think there is a "must…only" missing here.
```suggestion
		return "", errors.New("key must contain exactly 64 hex digits")
```

Reply:
🤦